src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
You can add **margin-right: 4px**; here and remove this .**stars :nth-child(n+2) {
  margin-left: 4px;
}**